tests/data/scalars_schema.json

```diff
@@ -1,33 +1,32 @@
 [
   {
     "mode": "NULLABLE",
-    "name": "timestamp_col",
-    "type": "TIMESTAMP"
+    "name": "bool_col",
+    "type": "BOOLEAN"
   },
   {
     "mode": "NULLABLE",
-    "name": "time_col",
-    "type": "TIME"
+    "name": "bignumeric_col",
+    "type": "BIGNUMERIC"
   },
   {
     "mode": "NULLABLE",
-    "name": "float64_col",
-    "type": "FLOAT"
+    "name": "bytes_col",
+    "type": "BYTES"
   },
   {
     "mode": "NULLABLE",
-    "name": "datetime_col",
-    "type": "DATETIME"
+    "name": "date_col",
+    "type": "DATE"
   },
   {
     "mode": "NULLABLE",
-    "name": "bignumeric_col",
-    "type": "BIGNUMERIC"
+    "name": "datetime_col", "type": "DATETIME"
```

Comment:
Didn't black complain about these two dict items on the same line? :thinking:

Reply:
I don't think we have a linter for JSON files configured, but good catch.

Reply:
Ah... it's a JSON file, overlooked that when scrolling up and down. :upside_down_face: